contracts/v0.2/README.md

```diff
@@ -39,3 +39,135 @@ cp .env.example .env
 ```shell
 forge script DeployFull --broadcast --rpc-url [RPC]
 ```
+
+## Migration Examples
+
+### Deploy `Feed` & `FeedProxy` Contracts
+
+```
+{
+  "Feed": {
+    "deploy": {
+      "submitter": "0xa195bE68Bd37EBFfB056279Dc3d236fAa6F23670",
+      "count": 2,
+      "feeds": [
+        { "decimals": 8, "description": "ADA-USDT" },
+        { "decimals": 8, "description": "ATOM-USDT" }
+      ]
+    }
+  }
+}
+```
+
+### Update Submitter of `Feed` Contract
+
+```
+{
+  "Feed": {
+    "updateSubmitter": {
+      "submitter": "0xa195bE68Bd37EBFfB056279Dc3d236fAa6F23670",
+      "count": 2,
+      "feeds": [
+        "0xc765f5ed9abb26349054020feea04f955a5cb1ec",
+        "0x9bb8f7b9f08ecc75aba62ba25d7b3f46fce79745"
+      ]
+    }
+  }
+}
+```
+
+### Deploy `FeedRouter`
+
+```
+{
+  "FeedRouter": {
+    "deploy": {}
+  }
+}
+
+```
+
+### Update proxies in `FeedRouter`
+
+```
+{
+  "FeedRouter": {
+    "address": "0x1ac6cd893eddb6cac15e5a9fc549335b8b449015",
+    "updateProxyBulk": {
+      "count": 2,
+      "proxies": [
+        {
+          "feedName": "BTC-USDT",
+          "proxyAddress": "0x50c23983ea26f30d368da5b257001ee3ddf9a539"
+        },
+        {
+          "feedName": "KLAY-USDT",
+          "proxyAddress": "0xd07bd0bcd3a8fa1087430b1be457e05c4a412a4b"
+        }
+      ]
+    }
+  }
+}
+```
+
+### Deploy `SubmissionProxy` and Register Oracle
+
+```
+{
+  "SubmissionProxy": {
+    "deploy": {},
+    "addOracle": {
+      "count": 1,
+      "oracles": ["0x50c23983ea26f30d368da5b257001ee3ddf9a539"]
+    },
+  }
+}
+```
+
+### Update Configuration of `SubmissionProxy`
+
+```
+{
+  "SubmissionProxy": {
+    "address": "0x1ac6cd893eddb6cac15e5a9fc549335b8b449015",
+    "setMaxSubmission": 120,
+    "setDataFreshness": 2,
+    "setExpirationPeriod": 2592000,
+    "setDefaultProofThreshold": 80,
+    "setProofThreshold": {
+      "count": 1,
+      "thresholds": [
+        {
+          "feed": "0xd07bd0bcd3a8fa1087430b1be457e05c4a412a4b",
+          "threshold": 60
+        }
+      ]
+    },
+    "addOracle": {
+      "count": 1,
+      "oracles": ["0x50c23983ea26f30d368da5b257001ee3ddf9a539"]
+    },
+    "removeOracle": {
+      "count": 1,
+      "oracles": ["0xd07bd0bcd3a8fa1087430b1be457e05c4a412a4b"]
+    }
+  }
+}
+```
+
+### Deploy `SubmissionProxy` with `Feed` and `FeedProxy` contracts
+
+```
```

Comment:
@nick-bisonai I made update and your comment disappeared.

The previous version did not seem finished. Probably, I pushed half completed version. I want to include there the word deploy to explain that it will deploy. Currently, I am thinking to deploy through this both `Feed` and `FeedProxy`. I am expecting this configuration to be used for the initial main deployment.

Reply:
understood :)